extension/src/background/helpers/dataStorage.ts

```diff
@@ -0,0 +1,36 @@
+import { browser } from "webextension-polyfill-ts";
```

Comment:
dumb questions alert 🙂 : 
- is the idea for using webextension-polyfill for the background storage that it's using [chrome storage](https://developer.chrome.com/docs/extensions/reference/storage/) for chrome and firefox storage (whatever that is) for firefox? I tried glancing through the [repo](https://github.com/mozilla/webextension-polyfill) but didn't see where storage was defined
- we won't have to change localstorage in the extension right? the manifest v3 changes are only for changes of background script -> service worker right?

Reply:
Not dumb questions!

You're exactly right on both questions 😄